### PR TITLE
#441 test storybook fix

### LINKED ISSUE #441: Fix tests and storybook
The tests and storybook are both failing due to the introduction of `i18next` which is complaining about `react` being undefined.

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #441 
This gets storybook working again! 
The tests now pass too, though I'm disappointed at having to mock i18next - was having trouble getting it working within the test environment and eventually decided that having components translated wasn't core the test cases

### CODE REVIEW

File: __mocks__/react-i18next.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+const reactI18Next: any = jest.createMockFromModule('react-i18next');
+
+reactI18Next.useTranslation = () => {
```
Comment (mark-prins) [inline __mocks__/react-i18next.ts:4]: Either `I18nextProvider(...): Nothing was returned from render` or getting `undefined` from `react`.. so.. mocks!

File: packages/common/src/intl/IntlProvider.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { I18nextProvider, initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
 
 const defaultNS = 'common';
-export const IntlProviderNext: React.FC = ({ children }) => {
+export const IntlProvider: React.FC = ({ children }) => {
```
Comment (mark-prins) [inline packages/common/src/intl/IntlProvider.tsx:10]: oops. had forgotten to rename this when the original was removed

File: packages/common/src/utils/testing.tsx [hunk 3]
```diff
@@ -36,14 +35,23 @@ export const TestingRouter: FC<TestingRouterProps> = ({
   </MemoryRouter>
 );
 
-export const TestingProvider: FC<TestingProviderProps> = ({
-  children,
-  locale = 'en',
-}) => (
+export const TestingProvider: FC = ({ children }) => (
```
Comment (mark-prins) [inline packages/common/src/utils/testing.tsx:38]: removed the provider for the tests, and use the mock instead.

**Review by joshxg — APPROVED** [review]
Nice! 🎉

**Pull request merged**